app/models/contract.rb

```diff
@@ -0,0 +1,7 @@
+class Contract < ApplicationRecord
+  belongs_to :statement
+  belongs_to :course
+  belongs_to :contract_template
+
+  validates :course_id, uniqueness: { scope: :statement_id, message: "Can only have one contract for statement and course" }
```

Comment:
Can we stick this in the translations file? ditto for other validation messages

Reply:
The problem with using locales is it's much more difficult to spot when one's missing and we end up with `"is invalid"` errors. We're just going to avoid locales altogether in ECF2.

Harder here though as there's already a mix of approaches.

Reply:
True, but there should be tests ensuring the messages are present/correct; having them in the model just adds clutter and IMO they're easier to digest/keep DRY in the locales. But that's just my opinion 🙂

Reply:
Moved to the translation file 👍